x-pack/plugins/maps/public/connected_components/mb_map/tooltip_control/tooltip_control.tsx

```diff
@@ -178,7 +178,7 @@ export class TooltipControl extends Component<Props, {}> {
     for (let i = 0; i < mbFeatures.length; i++) {
       const mbFeature = mbFeatures[i];
       const layer = this._getLayerByMbLayerId(mbFeature.layer.id);
-      if (!layer) {
+      if (!layer || !layer.canShowTooltip() || layer.areTooltipsDisabled()) {
```

Comment:
Your changes highlight an existing bug in `_getTooltipFeatures`. `continue` should be used instead of `break`. Would you mind fixing this?

To view the bug:
1) create a map
2) add 2 clusters layer for the same source
3) mouse over any cluster, notice how there are 2 features (as expected)
4) edit the top layer, and uncheck "Show tooltips"
5) mouse over any cluster, notice how no tooltips are displayed. We should get tooltips for the bottom layer. Thats because `break` exits the for loop for the feature from the  top layer so the feature from the bottom layer never gets processed.

Reply:
Good catch!